watcher_trigger.go

```diff
@@ -56,6 +57,19 @@ type trigger interface {
 	IsStop(n interface{}, err error) bool
 }
 
+// trgWatched is a the base data structure representing watched file/directory.
+// The platform specific full data structure (watched) must embed this type.
+type trgWatched struct {
```

Comment:
/cc @pblaszczyk

Reply:
I think Pawel's review is not needed here since this struct is just a "base" for `watched` type from  Kqueue's and FEN impls.